include/powsybl/test/XmlFixture.hpp

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright (c) 2019, RTE (http://www.rte-france.com)
```

Comment:
I noticed some 2019 and 2020, maybe 2019-2020 ?

Reply:
For now we do not consider date ranges, but only the starting date. We are not sure what is the best practice for that, but this should be done in a different PR, if we decide to do it.